src/commands/init.js

```diff
@@ -13,7 +11,7 @@ const makeConfig = require( './init/config' );
 exports.command = 'init';
 exports.desc = 'Creates a new configuration file to use for a project.';
 
-exports.handler = makeCommand( chalk, logSymbols, async () => {
+exports.handler = makeCommand( {}, {}, async () => {
```

Comment:
Docker is not needed for this command, let's not check for it and use: `checkDocker: false`